Cards/MadGraph5_aMCatNLO/NewTests/QCD-4Jets_HT-1000to1200_madgraphMLM-BGenFilter-pythia8/QCD-4Jets_HT-1000to1200_madgraphMLM-BGenFilter-pythia8.json

```diff
@@ -0,0 +1,18 @@
+{
+    "gridpack_submit": true,
+    "gridpack_path" : "QCD/QCD-4Jets_HT-1000to1200_madgraphMLM-pythia8",
+    "fragment": ["Generator/ExternalLHEProducer_MadGraph5_aMCatNLO.dat", "PartonShower/madgraphMLM-hepmcfilter-pythia8.dat", "Filter/LHE_BGenFilterEQ0_filter.dat"],
+    "fragment_user": [],
+    "fragment_vars": {
+        "processParameters": [
+            "'JetMatching:qCut = 14.'",
+            "'JetMatching:nQmatch = 5'",
+            "'JetMatching:nJetMax = 4'",
+            "'TimeShower:mMaxGamma = 4.0'"
+        ]
+        "hepmcFilter" : [
+            "nAttempts = cms.uint32(22),\nHepMCFilter = cms.PSet(\nfilterName = cms.string('PartonShowerBsHepMCFilter'),\nfilterParameters = cms.PSet())" 
```

Comment:
is it not possible to split the lines with commas like processParameters instead of writing everything in one go?

also we have to check
1. If "nAttempts = cms.uint32(22)" this is well understood
2. can we just have this fixed to certain number in the template and really keep the part starting from "HepMCFilter" only in the json file?

Reply:
@agrohsje any thoughts on this?

Reply:
As far as I saw we are only using 22. But I would make this an option at the json level.